CS342MemberViews/Sources/BrianPark/Github.swift

```diff
@@ -0,0 +1,42 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+struct Github: View {
+    @Environment(\.openURL) var openURL
+
+
+    var body: some View {
+        if let GitURL {
```

Comment:
The variable name should start with a lowercase letter.

```suggestion
        if let gitURL {
```